pdsfile/pds4file/tests/test_pds4file_blackbox.py

```diff
@@ -55,6 +49,78 @@ class TestPds4FileBlackBox:
+            ('uranus_occs_earthbased/uranus_occ_u0_kao_91cm/data/rings/u0_kao_91cm_734nm_radius_epsilon_egress_100m.xml',
+             'kao0m91-vis-occ-1977-069-u0-epsilon-e'),
+            ('uranus_occs_earthbased/uranus_occ_u0_kao_91cm/data/rings/u0_kao_91cm_734nm_radius_eta_egress_500m.tab',
+             'kao0m91-vis-occ-1977-069-u0-eta-e'),
+            ('uranus_occs_earthbased/uranus_occ_u0_kao_91cm/data/rings/u0_kao_91cm_734nm_radius_eta_ingress_1000m.xml',
+             'kao0m91-vis-occ-1977-069-u0-eta-i'),
+            ('uranus_occs_earthbased/uranus_occ_u0_kao_91cm/data/rings/u0_kao_91cm_734nm_radius_eta_egress_100m.xml',
+             'kao0m91-vis-occ-1977-069-u0-eta-e'),
+            ('uranus_occs_earthbased/uranus_occ_u0_kao_91cm/data/rings/u0_kao_91cm_734nm_radius_five_egress_500m.xml',
+             'kao0m91-vis-occ-1977-069-u0-five-e'),
+            ('uranus_occs_earthbased/uranus_occ_u0_kao_91cm/data/rings/u0_kao_91cm_734nm_radius_five_egress_100m.tab',
+             'kao0m91-vis-occ-1977-069-u0-five-e'),
+            ('uranus_occs_earthbased/uranus_occ_u0_kao_91cm/data/rings/u0_kao_91cm_734nm_radius_five_ingress_1000m.tab',
+             'kao0m91-vis-occ-1977-069-u0-five-i'),
+            ('uranus_occs_earthbased/uranus_occ_u0_kao_91cm/data/rings/u0_kao_91cm_734nm_radius_four_egress_1000m.tab',
+             'kao0m91-vis-occ-1977-069-u0-four-e'),
+            ('uranus_occs_earthbased/uranus_occ_u0_kao_91cm/data/rings/u0_kao_91cm_734nm_radius_four_ingress_100m.xml',
+             'kao0m91-vis-occ-1977-069-u0-four-i'),
+            ('uranus_occs_earthbased/uranus_occ_u0_kao_91cm/data/rings/u0_kao_91cm_734nm_radius_four_ingress_500m.xml',
+             'kao0m91-vis-occ-1977-069-u0-four-i'),
+            ('uranus_occs_earthbased/uranus_occ_u0_kao_91cm/data/rings/u0_kao_91cm_734nm_radius_gamma_ingress_500m.tab',
+             'kao0m91-vis-occ-1977-069-u0-gamma-i'),
+            ('uranus_occs_earthbased/uranus_occ_u0_kao_91cm/data/rings/u0_kao_91cm_734nm_radius_gamma_egress_1000m.xml',
+             'kao0m91-vis-occ-1977-069-u0-gamma-e'),
+            ('uranus_occs_earthbased/uranus_occ_u0_kao_91cm/data/rings/u0_kao_91cm_734nm_radius_gamma_ingress_100m.tab',
+             'kao0m91-vis-occ-1977-069-u0-gamma-i'),
+            ('uranus_occs_earthbased/uranus_occ_u0_kao_91cm/data/rings/u0_kao_91cm_734nm_radius_six_ingress_100m.xml',
+             'kao0m91-vis-occ-1977-069-u0-six-i'),
+            ('uranus_occs_earthbased/uranus_occ_u0_kao_91cm/data/rings/u0_kao_91cm_734nm_radius_six_egress_1000m.tab',
+             'kao0m91-vis-occ-1977-069-u0-six-e'),
+            ('uranus_occs_earthbased/uranus_occ_u0_kao_91cm/data/rings/u0_kao_91cm_734nm_radius_six_egress_500m.tab',
+             'kao0m91-vis-occ-1977-069-u0-six-e'),
+            ('uranus_occs_earthbased/uranus_occ_u0_kao_91cm/data/rings/u0_kao_91cm_734nm_radius_six_ingress_100m.xml',
+             'kao0m91-vis-occ-1977-069-u0-six-i'),
```

Comment:
Similar as above, remove the test cases for `.*counts-v-time_occult.*` under `/global` and `/rings` since we don't need opus id for them